Update tutorial-simulate-device-linux.md

damonbarry opened the pull request:
Raspbian has some additional prerequisites, which are not documented here. They are documented on [pypi](https://pypi.python.org/pypi/azure-iot-edge-runtime-ctl), but are not very discoverable. We've had requests to add the information to this document so more people will see it.

azurecla commented:
Hi __@dhei__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (dihei). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

ggailey777 commented:
@dhei Thanks for helping us copyedit this new topic. (I guess we were in a hurry to get it published :blush:) We really appreciate your help improving the quality of Azure docs.

damonbarry committed:
Update tutorial-simulate-device-linux.md

Raspbian has some additional prerequisites, which are not documented here. They are documented on [pypi](https://pypi.python.org/pypi/azure-iot-edge-runtime-ctl), but are not very discoverable. We've had requests to add the information to this document so more people will see it.
